vocabularies/UI.xml

```diff
@@ -702,6 +702,40 @@ The trend is
         </Property>
       </ComplexType>
 
+      <Term Name="Grid" Type="UI.GridDefinitionType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="UI.ThingPerspective" />
+        <Annotation Term="Core.Description" String="Visualization of a data grid" />
+      </Term>
+      <ComplexType Name="GridDefinitionType">
+        <Property Name="Title" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Title of the grid" />
+          <Annotation Term="Core.IsLanguageDependent" />
+        </Property>
+        <Property Name="Description" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Short description" />
+          <Annotation Term="Core.IsLanguageDependent" />
+        </Property>
+        <Property Name="Columns" Type="Collection(UI.DataFieldGridAbstract)" Nullable="false">
+          <Annotation Term="Core.Description" String="Columns of the grid" />
+        </Property>
+        <Property Name="Rows" Type="Collection(UI.DataFieldGridAbstract)" Nullable="false">
+        </Property>
+        <Property Name="Actions" Type="Collection(UI.DataFieldForActionAbstract)" Nullable="false">
+          <Annotation Term="Core.Description" String="Available actions" />
+        </Property>
+      </ComplexType>
+      <ComplexType Name="DataFieldGridAbstract" BaseType="UI.DataFieldAbstract" Abstract="true">
+        <Annotation Term="Core.Description" String="Abstract type to restrict usage in Grid" />
+      </ComplexType>
+      <ComplexType Name="DataFieldMeasures" BaseType="UI.DataFieldGridAbstract">
+        <Annotation Term="Core.Description" String="A structured piece of data described by an annotation" />
+        <Property Name="Measures" Type="Collection(Edm.PropertyPath)" Nullable="false" />
+      </ComplexType>
```

Comment:
One `DataFieldMeasures` already contains several measures, and in addition to that you can have several `DataFieldMeasures`s in `UI.Grid/Columns`. Why these nested collections?

Reply:
That's not intented - thanks for the hint

Reply:
But you haven't changed it yet.

Reply:
there can be only one datafieldmeasures either in columns or in rows. I'm struggling with the definition of the rows in columns as there either can dimensions and measures there,  but measures can either be in the rows or in columns and there are always all together, dimensions can be spread over columns and rows. And of course the measure collection doesn't have a fixed position :-).

Reply:
> But you haven't changed it yet.

Because I had to think about it :-) - I'm not as fast as you.

Reply:
> > But you haven't changed it yet.
> 
> Because I had to think about it :-) - I'm not as fast as you.

And if you have an idea how to solve the dilemma, don't hesitate to publish it...

Reply:
> there can be only one datafieldmeasures either in columns or in rows

Sounds complicated. Do you have examples, ideally with visuals of the corresponding grids?

Reply:
I will send you (both) an email with a screen shot.

Reply:
> there can be only one datafieldmeasures either in columns or in rows. I'm struggling with the definition of the rows in columns as there either can dimensions and measures there, but measures can either be in the rows or in columns and there are always all together, dimensions can be spread over columns and rows. And of course the measure collection doesn't have a fixed position :-).

Let the `UI.Grid` have properties
* `Rows`, which can contain `UI.DataField`s with groupable values and `UI.ReferenceFacet`s pointing to `Analytics.StructureElements`
* `Columns`, which can contain the same things
* `Measures`, which can contain `UI.DataField`s with an aggregatable value
* a property `MeasuresAxis`, which is either `Rows` or `Columns`.